### PR TITLE
Fix subtitle view

### PR DESCRIPTION
**Pull request by abdelaziz-mahdy**

### CODE REVIEW

**Comment by JulienDev** [comment]
I'm using your solution in production since a few weeks, it works pretty well ❤️

**Comment by abdelaziz-mahdy** [comment]
> I'm using your solution in production since a few weeks, it works pretty well ❤️

Glad it worked ❤️

File: media_kit_video/lib/src/subtitle/subtitle_view.dart [hunk 1]
```diff
@@ -83,6 +83,10 @@ class SubtitleViewState extends State<SubtitleView> {
   /// {@macro subtitle_view}
   @override
   Widget build(BuildContext context) {
+    subtitle = widget.controller.player.state.subtitle;
```
[the remark this answers is not in the document]
Comment (abdelaziz-mahdy) [reply media_kit_video/lib/src/subtitle/subtitle_view.dart:86]: these allow new values to of the player to be used on build

[the remark this answers is not in the document]
Comment (abdelaziz-mahdy) [reply media_kit_video/lib/src/subtitle/subtitle_view.dart:86]: a better way to do it is just ignoring the local variable and just use `widget.controller.player.state.subtitle` for example 
what do you think?

[the remark this answers is not in the document]
Comment (moffatman) [reply media_kit_video/lib/src/subtitle/subtitle_view.dart:86]: Yeah don't cache it locally if you don't have to.

**Review by moffatman — APPROVED** [review]
LGTM

**Pull request merged**